CONTRIBUTING.md

```diff
@@ -1115,6 +1116,17 @@ The following linters are used by our codebase:
 
 - The command name **should not** have the data source explicit
 
+#### UI and UX
+
+<img width="1676" alt="Screenshot 2022-10-26 at 12 17 19" src="https://user-images.githubusercontent.com/25267873/198012768-4cfecf7b-e961-4e55-a613-6648c0107b1e.png">
```

Comment:
I would say that it would be clearer to have the image after the rules.
Something like:
"As illustrated in the example below:
<img .... > "